### PR TITLE
kjell 0.2.5

### PR DESCRIPTION
**Pull request by dunn**
The test was timing out for me, but I couldn't think of an alternative and maybe it will work for the bots.

### CODE REVIEW

**Comment by dunn** [comment]
Only worked on ML; did `open3` change?

File: Library/Formula/kjell.rb [hunk 1]
```diff
@@ -15,15 +17,14 @@ def install
   end
 
   test do
-    require 'open3'
-    Open3.popen3("#{bin}/kjell") do |stdin, stdout, _|
+    require "open3"
+    Open3.popen3("#{bin}/kjell") do |stdin, _, _|
```
Comment (mikemcquaid) [inline Library/Formula/kjell.rb]: Can maybe use `pipe_output` here?

Comment (dunn) [inline Library/Formula/kjell.rb]: Can't pipe commands into `kjell`, it only allows you to open the interactive shell and then input.

Comment (mikemcquaid) [inline Library/Formula/kjell.rb]: Ah, sucks.

**Comment by mikemcquaid** [comment]
> Only worked on ML; did open3 change between Ruby versions?

Shouldn't do but probably did.

File: Library/Formula/kjell.rb (hunk 1 above)
Comment (jacknagel) [inline Library/Formula/kjell.rb]: Can probably just use `IO.popen` if we're not doing anything with stdout/stderr?

**Comment by dunn** [comment]
karlll/kjell#25

**Comment by mikemcquaid** [comment]
Thanks @dunn. Happy to just strip down the test for now?

**Comment by dunn** [comment]
You mean just remove it? OK

**Comment by mikemcquaid** [comment]
Thanks @dunn!

**Pull request closed without merging**